src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineRewriter.StateMachineMethodToClassRewriter.vb

```diff
@@ -16,8 +17,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Inherits MethodToClassRewriter(Of TProxy)
 
             Protected Friend ReadOnly F As SyntheticBoundNodeFactory
-            Protected NextState As Integer = StateMachineStates.InitialIteratorState
-            Protected NextFinalizerState As Integer = StateMachineStates.FirstIteratorFinalizeState
+
+            Private ReadOnly _resumableStateAllocator As ResumableStateMachineStateAllocator
+            Private _nextFinalizerState As Integer = StateMachineStates.FirstIteratorFinalizeState
```

Comment:
>StateMachineStates.FirstIteratorFinalizeState [](http://example.com/codeflow?start=53&length=45)

Is this initializer value used? It looks like we set `_nextFinalizerState` in the constructor.

Reply:
You're right. Removed.